routes/login.routes.js

```diff
@@ -93,7 +84,7 @@ route.post('/userlogin',upload.single('profilepic'), function(req, res) {
       res.send("ERROR")
     })
     .then(function(use) {
-      ppResize(img,300,300)

```

Comment:
instead of passing 300 directly to the function, create a default parameter like default_size = 300 and pass that.

It will be better for understanding in future cases.

Reply:
Ok, let me change :)